mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB="$(mktemp --suffix -snapshot.db)"
+SNAPSHOT_SETTINGS_PATH="$(mktemp --suffix -snapshot_settings.py)"
```

Comment:
For what it's with, if you have opinions on the format of the tempfile (e.g. should be a valid Python module name), I like to use:

`mktemp snapshot_settings_XXXXXXXXXXX.py`

for example. See also `man mktemp` and look at the description of the `TEMPLATE` option.